packages/d2-app/package.json

```diff
@@ -1,18 +1,22 @@
 {
     "name": "@dhis2/cli-app",
-    "version": "0.11.0",
-    "description": "D2 CLI module for interacting with frontend applications",
-    "main": "index.js",
-    "repository": "https://github.com/dhis2/cli",
-    "author": "Austin McGee <austin@dhis2.org>",
+    "bin": {
+        "d2-app": "./bin/d2-app"
+    },
+    "version": "1.0.0",
+    "main": "src/index.js",
+    "author": "DHIS2 &lt;info@dhis2.org&gt;",
```

Comment:
This is a bug in the `create` template, need to un-escape it somehow

Reply:
But for now maybe manually is fine?

Reply:
86c2edd